content/hardware/04.pro/shields/portenta-vision-shield/tutorials/camera-to-bitmap-sd-card/content.md

```diff
@@ -80,7 +87,10 @@ First you need to include the needed libraries
 #include "FATFileSystem.h"    // Mbed API for portable  and embedded systems
 
 #include "camera.h" // Arduino Mbed Core Camera APIs
-#include "himax.h"  // API to read from the Himax camera found on the Portenta Vision Shield
+
+#include "himax.h"  // API to read from the Himax camera found on the Portenta Vision Shield Rev.1

```

Comment:
in the rest of the examples version 1 library is commented, should we do it here too?